gensim/models/fasttext.py

```diff
@@ -27,10 +30,88 @@
 
 .. sourcecode:: pycon
 
-    >>> from gensim.test.utils import common_texts
-    >>> from gensim.models import FastText
+    >>> # from gensim.models import FastText  # FIXME: why does Sphinx dislike this import?
+    >>> from gensim.test.utils import common_texts  # some example sentences
+    >>>
+    >>> print(common_texts[0])
+    ['human', 'interface', 'computer']
+    >>> print(len(common_texts))
+    9
+    >>> model = FastText(size=4, window=3, min_count=1)  # instantiate
+    >>> model.build_vocab(sentences=common_texts)
+    >>> model.train(sentences=common_texts, total_examples=len(common_texts), epochs=10)  # train
+
+Once you have a model, you can access its keyed vectors via the `model.wv` attributes.
+The keyed vectors instance is quite powerful: it can perform a wide range of NLP tasks.
+For a full list of examples, see :class:`~gensim.models.keyedvectors.FastTextKeyedVectors`.
+
+You can also pass all the above parameters to the constructor to do everything
+in a single line:
+
+.. sourcecode:: pycon
+
+    >>> model2 = FastText(size=4, window=3, min_count=1, sentences=common_texts, iter=10)
+
+.. Important::
+    We intend to deprecate this second method of passing everything through the constructor.
```

Comment:
```suggestion
    This style of initialize-and-train in a single line is **deprecated**. It's included for backward compatibility only. 
    Please use the initialize-build_vocab`-`train` pattern above instead, including using `epochs` instead of `iter`.
```